portfolio/src/main/webapp/script.js

```diff
@@ -64,6 +64,101 @@ function createListElement(text) {
   return liElement;
 }
 
+var mitMarker;
+var dormMarker; 
+
+var mitLatLng;
+var dormLatLng;
+
+var directionsRenderer;
+var directionsService;
+
+// Creating map
+function createMap() {
+    directionsRenderer = new google.maps.DirectionsRenderer();
+    directionsService = new google.maps.DirectionsService();
+
+    mitLatLng = {lat: 42.3601, lng: -71.0942};
+    dormLatLng = {lat: 42.355338, lng: -71.100530};
+
+    const map = new google.maps.Map(document.getElementById('portfolio-map'),
+        {
+            center: {
+                lat: (mitLatLng.lat+dormLatLng.lat)/2, 
+                lng: (mitLatLng.lng+dormLatLng.lng)/2
+            }, 
+            mapTypeId: 'terrain',
+            zoom: 16
+        }); 
+
+    directionsRenderer.setMap(map);
+    renderMap(map);
+}
+
+function renderMap(map) {
+    mitMarker = new google.maps.Marker({
+        position: mitLatLng,
+        map: map,
+        animation: google.maps.Animation.DROP,
+        title: 'MIT (My University)'
+    });
+
+    dormMarker = new google.maps.Marker({
+        position: dormLatLng,
+        map: map,
+        animation: google.maps.Animation.DROP,
+        title: 'New House'
+    });
+
+    campusContentString = "MIT is where I currently go to college." + 
+    "<br />It is located in Cambridge, Massachusetts."
```

Comment:
Is there supposed to be a `+` at the end of this line?
Also 4 spaces for continued new lines.

```
campusContentString = "MIT is where I currently go to college." +
    "<br />It is located in Cambridge, Massachusetts." +
    "<br />This is main campus.";
```
Apply here and in dormInfoString